guis/desktop/gui.py

```diff
@@ -46,12 +46,13 @@ class Geometry:
     type: str
     coors: list or tuple
 
+
     def flip(self):
         """
         Switch lat lon
         """
         corrected_coors = []
-        if self.type == 'Point':
+        if self.type == "Point":
             corrected_coors = self.coors[1], self.coors[0]
         else:
             self.coors = self.coors[0]
```

Comment:
Looking over it again, this line can be removed and use the following loop deeper into the array.